tests/test-cases/Group1-Docker-Commands/1-38-Docker-Exec.robot

```diff
@@ -63,7 +63,7 @@ Wait Until Detached Exec Occurs
      \   ${Status1}=  Run Keyword And Return Status  Should Be Equal As Integers  ${rc}  0
      \   ${Status2}=  Run Keyword And Return Status  Should Contain  ${output}  force
      \   Return From Keyword If  ${status1} & ${status2}
-     \   Sleep  1s
+     \   Sleep  2s
```

Comment:
Doesn’t the Wait Until Keyword Succeeds keyword do this same logic? Why do we have a dedicated loop in here?

Reply:
this was recommended by @mhagen-vmware during review.